skrf/plotting.py

```diff
@@ -82,6 +82,29 @@
 from . util import now_string_2_dt
 import matplotlib.tri as tri
 
+from functools import wraps
+    
+def check_plotting(matplotlib_pyplot_alias):
```

Comment:
A function like this should probably in __ini__.py or utils.py, otherwise we would not be able to run it.

Reply:
Moved to `util.py`